components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/workspace/workspace.js

```diff
@@ -193,6 +193,14 @@ define(['ace/ace', 'jquery', 'lodash', 'log','dialogs','./service-client','welco
                 }
             };
 
+            this.handleFormat = function () {
+                var activeTab = app.tabController.getActiveTab();
```

Comment:
No need of assigning this into a variable since the variable refers in only one place.

Reply:
Corrected in commit e30b1f4